tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -191,7 +192,7 @@ Clean Up User Notebook
 
       # Verify that the jupyter notebook pod is running
       ${notebook_pod_name} =   Get User Notebook Pod Name  ${username}
-      Search Pods    ${notebook_pod_name}  namespace=rhods-notebooks
+      OpenShiftCLI.Search Pods    ${notebook_pod_name}  namespace=rhods-notebooks
```

Comment:
It's better to use the new OpenShift Library instead of OpenShiftCLI

Reply:
Updated

Reply:
Updated